### PR TITLE
Update repo and author

### PR DESCRIPTION
**Pull request by martinvd**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,8 +2,13 @@
   "name": "download-ccleaner",
   "version": "0.1.1",
   "description": "Download and extract the latest CCleaner",
-  "author": "martinvd",
   "license": "MIT",
+  "repository": "martinvd/download-ccleaner",
+  "author": {
+    "name": "Martin van Driel",
+    "email": "martinvdpub@gmail.com",
```
Comment (winfried-van-loon) [inline package.json]: Perhaps use your public GitHub emailaddress, if you've got one?

File: package.json [hunk 2]
```diff
@@ -2,8 +2,13 @@
...
+    "url": "martinvandriel.com"
```
Comment (winfried-van-loon) [inline package.json]: http:// prefix? :)

**Review by winfried-van-loon — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (martinvd) [reply package.json]: Thanks! I added it 👍

File: package.json (hunk 1 above)
Comment (martinvd) [reply package.json]: You are right, thanks man! 😃

**Pull request merged**